budcamp/Homework3/Homework3/task2/TestResults.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework3
+{
+    internal class TestResults
+    {
+        private Dictionary<string, int> results;
+
+        public TestResults()
+        {
+            results = new Dictionary<string, int>();
+        }
+
+        public void AddResults(string topic, int mark)
```

Comment:
According to the class definition I provided above, this method will change to something like this:

```
public void AddResults(string topic, TestResult result) {
    results[topic] = result;
}
```